pytorch_lightning/trainer/ignored_warnings.py

```diff
@@ -3,11 +3,11 @@
 
 def ignore_scalar_return_in_dp():
     # Users get confused by this warning so we silence it
-    m_1 = """
```

Comment:
This warning message didn't match with the real one because triple quoted strings also include line breaks as \n.

Reply:
With this PR, why is it still necessary to have this warning filter?

Reply:
We could fix this by unsqueeze the 0-d tensors ourselves but I don't know if this is better than silencing the warning.

Reply:
ah I forgot, this is tensor scalars but your PR is about Python scalars. Ignore my comment

Reply:
Yeah, this is an unrelated issue :D